src/app/settings/views/Configuration/Security/VaultSettings/VaultSettings.test.tsx

```diff
@@ -0,0 +1,157 @@
+import { screen } from "@testing-library/react";
+
+import VaultSettings, { Labels as VaultSettingsLabels } from "./VaultSettings";
+
+import type { Controller } from "app/store/controller/types";
+import type { RootState } from "app/store/root/types";
+import { NodeType } from "app/store/types/node";
+import {
+  generalState as generalStateFactory,
+  controller as controllerFactory,
+  controllerState as controllerStateFactory,
+  rootState as rootStateFactory,
+  vaultEnabledState as vaultEnabledStateFactory,
+} from "testing/factories";
+import { renderWithMockStore } from "testing/utils";
+
+describe("VaultSettings", () => {
```

Comment:
👍 For maintaining a good testing coverage.

Reply:
Thanks!